README.md

```diff
@@ -50,7 +50,17 @@ The nf-core/pangenome pipeline comes with documentation about the pipeline which
 
 ## Credits
 
-nf-core/pangenome was originally written by Simon Heumos, Michael L Heuer.
+nf-core/pangenome was originally adapted from the pangenome graph builder [`pggb`](https://github.com/pangenome/pggb) pipeline by Simon Heumos, Michael Heuer.
+
+Many thanks to all who have helped out and contributed along the way, including (but not limited to)\*:
```

Comment:
I am not sure when to bring in the people who currently work on the related MultiQC report module https://github.com/Imipenem/MultiQC/tree/module/odgi_stats/multiqc/modules/odgi @Zethson @Imipenem.

Also, if we take the docker image or Dockerfile from https://github.com/pangenome/pggb, we might want to add @ekg @AndreaGuarracino

Reply:
👍 Please @ekg @AndreaGuarracino  @Zethson @Imipenem add additional lines to this pull request as additional commits, or make follow up pull requests

Reply:
Thanks @Zethson for adding everyone.